### DISCUSSION TITLE
Add password strength score indicator

### DISCUSSION THREAD

**Discussion by desiboli** [shadcn-ui/ui#762] [Ideas]
Hi!
Love the work you do! 🙏

Would be awesome if this feature could be added as an option for password input fields.

Here is a couple of examples of the feature I'm asking for:
[https://tailwindcomponents.com/component/password-generator-and-strength-score](https://tailwindcomponents.com/component/password-generator-and-strength-score)

[https://codyhouse.co/ds/components/app/password-strength-indicator](https://codyhouse.co/ds/components/app/password-strength-indicator)

[https://dribbble.com/shots/18399432-Wix-s-Password-Strength-Indicator](https://dribbble.com/shots/18399432-Wix-s-Password-Strength-Indicator)

[https://dribbble.com/shots/18325342-Set-New-Password](https://dribbble.com/shots/18325342-Set-New-Password)

**Comment by markhallak** [comment]
Any updates?

**Reply by Super-Kenil** [reply]
I guess not

**Reply by PoSayDone** [reply]
https://originui.com/input
Component "Input with password strength indicator"

**Reply by Super-Kenil** [reply]
This one's great, a one stop shop for all kind of inputs, including password strength meter

**Comment by Nima-sltn** [comment]
Try [RigidUI](https://www.rigidui.com/docs/components/strength-meter#usage) , one of Shadcn registries
